content/en/events/_index.md

```diff
@@ -78,11 +127,20 @@ sources:nagios,chef status:error cassandra
 
 **Note**: Do not use spaces after the colon or commas in these lists. Anything not attached to a prefix goes to full text search.
 
+{{< site-region region="us" >}}
+### Aggregation
+
+By default, related events are aggregated when displayed in the events stream. To show unaggregated events, un-check the **Aggregate related events** box at the top right of your event stream:
+
+{{< img src="events/event_stream_aggregated.png" alt="Aggregated event stream"  style="width:50%;" >}}
+{{< /site-region >}}
+{{< site-region region="eu" >}}
 ### Aggregation
 
 By default, related events are aggregated when displayed in the events stream. To show unaggregated events, un-check the **Aggregate related events** box at the top right of your event stream:
 
 {{< img src="events/event_stream_aggregated.png" alt="Aggregated event stream"  style="width:50%;" >}}
+{{< /site-region >}}
 
 ### Notifications
```

Comment:
I thought these weren't supported in GovCloud? The Event Management team can confirm though.